Sources/Common/DataModel/DataSetAttributes/FieldData.d.ts

```diff
@@ -6,7 +6,7 @@ import vtkDataArray from "../../Core/DataArray";
  * 
  */
 export interface IFieldDataInitialValues {
-	arrays?: Array<any>;
+	arrays?: Array<vtkDataArray> | vtkDataArray[];
```

Comment:
`vtkDataArray[]` is the short form of `Array<vtkDataArray>` , so you can just use `vtkDataArray[]`